website/docs/components/rich-tooltip/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,24 @@
+## Conformance rating
+
+<!-- Update conformance rating badge with correct status and remove the others -->
+<Doc::Badge @type="success">Conformant</Doc::Badge>
```

Comment:
I think we need to say conditionally conformant. Here's why. 

On the small and medium heights, the design system itself cannot guarantee that the consumer has left enough space around, and just on its own it does not meet the minimum size requirements. 

Additionally, we can't check for this because when used inside of a sentence or paragraph of text, it's exempt from that rule (since there is no rule saying that there needs to be a minimum line-height). 

These are the reasons I think we should state that it's conditionally conformant, and give the user these details. WDYT?

Reply:
This should be addresed as part of https://hashicorp.atlassian.net/browse/HDS-3294, yes?